Assets/Scripts/FpsController.cs

```diff
@@ -0,0 +1,158 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FpsController : MonoBehaviour
+{
+    public Rigidbody body;
+
+    [Header("Camera Settings")]
+    public Camera cam;
+    public float lookHorizontalSensitivity = 1.0f;
+    public float lookVerticalSensitivity = 1.0f;
+    public float minimumPitch = -89.9f;
+    public float maximumPitch = 89.9f;
+
+    [Header("Jump Settings")]
+    public float jumpForce = 300.0f;
+    public float jumpInputBufferTime = 0.25f;
+
+    [Header("Movement Settings")]
+    [Range(0.1f, 15.0f)]
+    public float forwardSpeed = 5.0f;
+    [Range(0.1f, 15.0f)]
+    public float backSpeed = 3.0f;
+
+    private Vector2 _inputVector = Vector2.zero;
+    private bool _jumpInput = false;
```

Comment:
I personally like to use the prefixes "is" or "has" for booleans

Suggest renaming this variable from `_jumpInput` to `_isJumpKeyPressed`